src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -133,9 +134,21 @@ const PartnerDataButtons = ({ partner, editPartner }: PartnerViewEditProps) => (
       label="Status"
       onClick={() => editPartner('active')}
       secured={partner?.active}
-      startIcon={<TimelineIcon color="info" />}
+      startIcon={
+        partner?.active.value ? (
+          <TimelineIcon color="info" />
+        ) : (
+          <InactiveStatusIcon color="error" />
+        )
+      }
       redacted="You do not have permission to view Status"
-      children={partner?.active.value ? 'Active' : 'Inactive'}
+      children={
+        partner?.active.value ? (
+          'Active'
+        ) : (
+          <Box color="text.secondary">Inactive</Box>
+        )
+      }
```

Comment:
I would do this to keep things a bit cleaner and easier to read:
```suggestion
      children={
        <Box color={partner?.active.value ? 'inherit' : 'text.secondary'}>
          {partner?.active.value ? 'Active' : 'Inactive'}
        </Box>
      }
```